src/SFA.DAS.Tools.Servicebus.Support.Web/appsettings.json

```diff
@@ -10,5 +10,7 @@
   "ConfigNames": "SFA.DAS.Tools.Servicebus.Support",
   "EnvironmentName": "LOCAL",
   "Version": "1.0",
-  "APPINSIGHTS_INSTRUMENTATIONKEY": ""
+  "APPINSIGHTS_INSTRUMENTATIONKEY": "",
+  "DefaultCosmosOperationTimeout": 55,
```

Comment:
should new settings go into das-employer-config?

Reply:
I've been working on the basis of not using employer config for simple keys that could be added to the pipeline and handled  by the web-deployment.yml that created the settings for the containers in Docker containers. This was done for the Api config for commitments in the tool currently deployed to test so was following that process. I can see that employer config has been used for the service bus support tool. Worth a chat on this i think?

Reply:
Have confirmed with Nathan & Witek. This tools config is shared due to deploying function app and to the KCluster.